source/_posts/2020-11-12-android-300-release.markdown

```diff
@@ -0,0 +1,128 @@
+- WiFi BSSID - This sensor has a setting that will allow the user to rename the currently connected BSSID. Not everyone can remember a mac address let alone dozens of them. This setting is designed to help out users who use this sensor to make better sense of things without the need for secrets or templates. If you live in a household with multiple access points you may find it useful to rename a BSSID to help with things like room presence. By default this sensor reports the connected mac address.
+
+
+## Android 11 Power Menu
+
+We now integrate with Android 11's power menu device control feature. The following domains are currently supported:
+
+-  `automation` On/Off
+-  `climate` Temperature slider
+-  `cover` Open/Close
+-  `fan` On/Off, speed slider
+-  `input_boolean` On/Off
+-  `input_number` Number control slider
+-  `light` On/Off, Brightness control slider
+-  `lock` Lock/Unlock
+-  `scene` Turn on scene
+-  `script` Turn on script
+-  `switch` On/Off
+
+
+<p class='img'>
+<img src='/images/blog/2020-11-05-android-300-release/power_menu.png' alt='Screenshot of power menu'></a>
+Screenshot of power menu.
+</p>
+
+
+## Notification Improvements
+
+There have been several improvements to notifications as well.
+
+- An event gets sent upon a notification being dismissed along with all notification data.
+- Notifications can make use of the alarm stream to bypass a device's ringer mode setting. This can be useful if there is an important event such as an alarm being triggered.
+- Text To Speech notifications, with the ability to use the alarm stream if desired. By default it will use the device's music stream. There is also an additional option to temporarily change the volume level to the maximum level while speaking, the level would then restored to what it was previously.
+- New device commands to control your phone: broadcasting an intent to another app, controlling Do Not Disturb and ringer mode.
+- Opening another app with an actionable notification.
+
+
+## Other Enhancements
+
+We have also spent time making improvements to all other areas most notably inside App Configuration:
+
+<p class='img'>
+<img src='/images/blog/2020-11-05-android-300-release/settings.png' alt='Screenshot of settings'></a>
+Screenshot of settings.
+</p>
+
+- App language can now be overridden to match a users profile, this will impact the name of entities that get added as well as the App Configuration appearance. By default your device will use your phones language however, we noticed some users actually prefer Home Assistant to always be in English. With this feature you can now set the language to be one of any that are supported by the app.
```

Comment:
```suggestion
- App language can now be overridden to match a user's profile, this will impact the name of entities that get added as well as the App Configuration appearance. By default your device will use your phones language however, we noticed some users actually prefer Home Assistant to always be in a different language to their device. With this feature you can now set the language to be one of any that are supported by the app.
```

Just remove the English bias. I link to Lokalize and a "want to help translate the app" bit here might be a good idea.

Reply:
Great suggestion!